provisioning/osc-provision

```diff
@@ -63,20 +97,54 @@ do
     --master-is-node)
       master_is_node=true;
       shift;;
+    --no-install)
+      no_install=true;
+      shift;;
     *)
-      echo "Invalid Option: ${i#*=}"
+      echo "Invalid Option: ${i%=*}"
       exit 1;
       ;;
   esac
 done
 
 if [ -z $num_of_nodes ]; then
-  echo "Missing argument: --num_of_nodes <integer>"
+  echo "Missing argument: --num-nodes <integer>"
   usage
   exit 1;
 fi
 
-${SCRIPT_BASE_DIR}/osc-sync-keys -m=${MASTER#*|} -n=${NODES#*|}
-${SSH_CMD} root@${MASTER#*|} "rm -rf ~/ose-utils"
-${SSH_CMD} root@${MASTER#*|} "git clone https://github.com/etsauer/ose-utils.git --branch openshift-enterprise-3 ~/ose-utils"
-${SSH_CMD} root@${MASTER#*|} "bash ~/ose-utils/provisioning/osc-install -m=\"${MASTER}\" -n=\"${NODES}\" -a=dns"
+# Set up environment based on default values and User input
+SSH_CMD='ssh -tt -o ServerAliveCountMax=100 -o ConnectionAttempts=180 -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -o LogLevel=ERROR'
```

Comment:
Yes, new file perhaps? 'global_variables', 'constants' ? Not sure yet what else might go in there